### PR TITLE
Create pull_request_template.md

### PR DESCRIPTION
**Pull request by beckybell13**

### CODE REVIEW

File: pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+- [ ] **Sensitive Data Exposure:** Is there any sensitive data that may be exposed with this change, such as emails or passwords? Do you have protections in place for that data?
+- [ ] **XML External Entities:** Do these changes involve processing or uploading XML, particularly from untrusted sources?
+- [ ] **Broken Access Control:** Have you added auth checks and middleware where needed and at the appropriate level? Are you filtering by the correct `scope`, `scope_id`, and `organization_id`?
+- [ ] **Security Misconfiguration:** Are you using a third-party library with default configurations? Do those defaults have a security risk?
+- [ ] **XSS:** Does this change involve using untrusted data to render HTML or execute Javascript? Have you properly validated/escaped it?
+- [ ] **Insecure Deserialization:** Are you using native (de)serialization formats? Can you use a JSON format instead? Does (de)serialization safely handle unexpected/malformed data?
+- [ ] **Using Components with Known Vulnerabilities:** Are you using a library, framework, or other third-party module with known vulnerabilities? Are existing modules up-to-date?
```
Comment (jantonso) [inline pull_request_template.md]: Maybe we can add something in here about running the `npm audit` command?

File: pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+- [ ] **Insufficient Logging and Monitoring:** Is there a way to monitor these changes so we can easily detect a breach/attack/irregular behavior?
+
+### Accessibility
+- [ ] Are you using semantic HTML? If you are unable to, does your custom widget have the appropriate aria attributes?
+- [ ] Is all functionality keyboard accessible?
+- [ ] Do all forms inputs have labels?
+- [ ] Do colors have sufficient contrast? Is text large enough? Refer to the WCAG 2.0 guidelines.
```
Comment (jantonso) [inline pull_request_template.md]: Can we link to this page: https://dequeuniversity.com/rules/axe/4.1/color-contrast?application=axeAPI or whatever other page everyone else has been using?

**Comment by jantonso** [comment]
Looks amazing!

How will this work? Do we choose the template when we create a PR?

**Comment by beckybell13** [comment]
I think it should auto-populate the PR description, but need try it out! This should be the default across all repos unless you add an override in the repo itself.

**Pull request merged**